packages/components/src/confirm-dialog/test/index.tsx

```diff
@@ -142,6 +142,17 @@ describe( 'Confirm', () => {
 
 				const confirmDialog = screen.getByRole( 'dialog' );
 
+				// TODO: Is there a better way you'd like this checked?
```

Comment:
@ciampo Lmk how you'd prefer this null check. Looks like there are a few eslint rules for this.

Reply:
Probably a quicker way would be to force TypeScript to consider `parentElement` non-null by adding a `!` at the end of the variable

`await user.click( confirmDialog.parentElement! );`

This way, we can keep the amount of changes in the file down to a minimum. The test will fail anyway in case `parentElement` is `null`, so I don't see issues with the approach